src/igvfd/schemas/measurement_set.json

```diff
@@ -197,6 +198,16 @@
                 "appscript": "[\"john-doe:auxiliary_set01\", \"TSTDS98084731\"]",
                 "igvf_utils": "john-doe:auxiliary_set01, TSTDS98084731"
             }
+        },
```

Comment:
Do we want to consider using a default value here to make submission simpler, since we only have 1 enum available?